cmd/reviewdog/main_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"testing"
 )
 
+var _ = fmt.Sprintf("", 1, 2, 3)
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
no formatting directive in Sprintf call